sceptre/scipool/config/develop/sc-portfolio-scheduled-jobs.yaml

```diff
@@ -0,0 +1,8 @@
+template_path: "sc-portfolio-scheduled-jobs.yaml"
+stack_name: "sc-portfolio-scheduled-jobs"
+dependencies:
+  - "develop/sc-lambda-launchrole.yaml"
```

Comment:
Should this be "develop/sc-scheduled-jobs-launchrole.yaml"?

Reply:
yes, fixed